src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "destKey": "channels.email.address",
+    "sourceKeys": "emailOnly",
```

Comment:
@Gauravudia  how will `emailOnly` gets instrumented outside `traits` ?

Reply:
It is part of  "sourceFromGenericMap": true

Reply:
https://github.com/rudderlabs/rudder-transformer/blob/20aa7f35e13ad89e8a43fbbb743df73b0c103975/src/v0/util/data/GenericFieldMapping.json#L6